runtime/Cargo.toml

```diff
@@ -65,11 +65,18 @@ notify = "5.0.0-pre.7"
 percent-encoding = "2.1.0"
 regex = "1.4.3"
 ring = "0.16.20"
+rustls = { version = "0.19.0", features = ["dangerous_configuration"] }
 serde = { version = "1.0.125", features = ["derive"] }
+serde_with = "1.9.3"
 sys-info = "0.9.0"
 termcolor = "1.1.2"
 tokio = { version = "1.8.0", features = ["full"] }
 uuid = { version = "0.8.2", features = ["v4"] }
+webpki = "0.21.4"
+webpki-roots = "0.21.1"
```

Comment:
Some dependencies are moved somewhere else, may be not required.

Reply:
You can remove them completely from this file - they are never used outside `extensions/` crates

Reply:
Done, there are also dependencies which have been not added by this PR but are unused. Should I remove them too?

Reply:
That's okay, but please do that in a separate PR